src/main/java/org/sonar/plugins/stash/StashIssueReportingPostJob.java

```diff
@@ -66,17 +61,14 @@ public void executeOn(Project project, SensorContext context) {
   * Second part of the code necessary for the executeOn() -- squid:S134
   */
   private void updateStashWithSonarInfo(StashClient stashClient, StashCredentials stashCredentials,
-                                                Project project, SensorContext context) {
+                                        SensorContext context) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "context". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)